cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/SlowBrokerFinder.java

```diff
@@ -416,6 +437,21 @@ public void configure(Map<String, ?> configs) {
       }
     }
 
+    String logFlushTimeThresholdMs = (String) originalConfig.get(SLOW_BROKER_LOG_FLUSH_TIME_THRESHOLD_MS_CONFIG);
```

Comment:
This function (i.e. configure) is full of repeated code fragments. Can we use util functions to eliminate code repetition and be more concise?